moto/mq/models.py

```diff
@@ -28,15 +28,13 @@ def __init__(
         data: Optional[str] = None,
     ):
         self.configuration_id = configuration_id
-        self.created = unix_time()
+        self.created = utcnow()
```

Comment:
Backend models should just use native Python `datetime` objects.  Proper formatting will be handled by the serializer.  (The serializer is forgiving, though, so a timestamp should still work if it's a string, which gives some wiggle room if some are missed when migrating services to the new serialization.)